### DISCUSSION TITLE
HOMA protocol support

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [netty/netty#13830] [Q&A]
Hi guys.

HOMA protocol tail latency is 10 times better than TCP and there is a standalone Java implementation of this protocol https://github.com/PlatformLab/grpc_homa

So I am curious: do you have any plans to implement it?

**Comment by chrisvest** [comment]
Maybe when/if HOMA becomes available in mainstream kernels. Meanwhile, experimental transports are possible to implement outside of Netty, for the adventurous.
